source/docs/v3/tenants.blade.md

```diff
@@ -130,6 +130,23 @@ You may access the current tenant using the `tenant()` helper. You can also pass
 
 Alternatively, you may typehint the `Stancl\Tenancy\Contracts\Tenant` interface to inject the model using the service container.
 
+## Accessing the central application {#accessing-the-central-application}
+
+When your code executes in the context of a tenant (for instance, when the `DatabaseMigrated` event is dispatched), you may access the central application context by using the `tenancy()->central($callback)` function.
+
+For instance, if you want to retrieve the list of users of the central application when the code executes in a tenant's context, you may write:
+
+```php
+public function getCentralUsers()
+{
+    // here we are in the tenant's context. \App\Models\User::all() would return the current tenant's users.
+    return tenancy()->central(function () {
+        // here we are in the central app context
+        return \App\Models\User::all();
+    });
```

Comment:
Maybe this could be like `$centralUsers = ...`? That would demonstrate nicely that the value can be worked with outside the closure, and it'd add clarity after the wrapper method is removed.

Reply:
Thanks for you inputs. I'm not used to contribute, should I close this pull request and reopen another one after I applied the changes? What's more I'm not fluent in english, so if you have any suggestions about phrasing, they're welcome.

Reply:
You can make the changes in this PR, just add more commits to your branch, or make the changes directly on GitHub by editing the file.